src/containers/Application/ApplicationList.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react'
+import { Container, Table, List, Label } from 'semantic-ui-react'
+import { Loading, FilterList } from '../../components'
+import { useRouter } from '../../utils/hooks/useRouter'
+import useListApplication from '../../utils/hooks/useListApplications'
+
+const ApplicationList: React.FC = () => {
+  const { query } = useRouter()
+  const { error, loading, applications } = useListApplication()
+
+  return error ? (
+    <Label content="Problem to load applications list" error={error} />
+  ) : loading ? (
+    <Loading />
+  ) : (
+    <Container>
+      <FilterList />
+      {Object.keys(query).length > 0 && <h3>Query parameters:</h3>}
+      <List>
+        {Object.entries(query).map(([key, value]) => (
+          <List.Item key={`app_selected_parameter_${value}`} content={key + ' : ' + value} />
+        ))}
+      </List>
+      {applications && applications.length > 0 && (
+        // TODO: Implement sortable columns -> Requires reducer
+        <Table sortable stackable selectable>
+          <Table.Header>
+            <Table.Row>
+              {Object.entries(applications[0]).map(([key]) => (
+                <Table.HeaderCell key={`app_header_${key}`}>{key}</Table.HeaderCell>
```

Comment:
I'm getting this error in the console:
```
Warning: Encountered two children with the same key, `app_2_null`. 
```
Is there a problem getting `null` values for some of these keys?

Reply:
I think I know what's happening. The default data has no stage or status for the "Company Registration", so these cells are getting `null` as part of their key.

Maybe we should assume that any application with no stage or status is erroneous, and therefore we shouldn't show it in this list?

Reply:
Ah. Yep, that's a good solution. I'll work on it now.

Reply:
Hmm, sorry to walk back on this now, but I think the [solution below](https://github.com/openmsupply/application-manager-web-app/pull/107#discussion_r520173067) is probably better. That way we can still list applications with no stage or status, without causing `null` keys.